.gitattributes

```diff
@@ -1,2 +1,4 @@
+*.go -text diff=golang
```

Comment:
git understands golang line endings by itself? That's awesome if so

Reply:
Yup yup!